src/props.shadows.js

```diff
@@ -38,23 +39,33 @@ const Shadows = {
     0 41px 33px -2px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 6%)),
     0 100px 80px -2px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 7%))`,
 
-  '--inner-shadow-0': 'inset 0 0 0 1px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-1': 'inset 0 1px 2px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-2': 'inset 0 1px 4px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-3': 'inset 0 2px 8px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--inner-shadow-4': 'inset 0 2px 14px 0 hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-}
+  "--inner-shadow-0":
+    "inset 0 0 0 1px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))",
+
+  "--inner-shadow-1": `
+    inset 0 -.5px 0 0 #fff1`,
+  "--inner-shadow-2": `
+    inset 0 -.5px 0 0 #fff1`,
+  "--inner-shadow-3": `
+    inset 0 -.5px 0 0 #fff1`,
+  "--inner-shadow-4": `
+    inset 0 -.5px 0 0 #fff1`,
+};
 
-export default Shadows
+export default Shadows;
 
 export const StaticShadows = Object.fromEntries(
   Object.entries(Shadows)
-    .map(([key,value]) => {
-      value = value.replace(/var(--shadow-color)/g, Shadows['--shadow-color'])
-      value = value.replace(/var(--shadow-strength)/g, Shadows['--shadow-strength'])
-      return [key, value]
+    .map(([key, value]) => {
+      value = value.replace(
+        /var\(--shadow-color\)/g,
+        Shadows["--shadow-color"]
+      );
+      value = value.replace(
+        /var\(--shadow-strength\)/g,
+        Shadows["--shadow-strength"]
+      );
+      return [key, value];
     })
-    .filter(([key]) =>
-      !['--shadow-color', '--shadow-strength'].includes(key)
-    )
-)
+    .filter(([key]) => !["--shadow-color", "--shadow-strength"].includes(key))
+);
```

Comment:
formatter changes again? this should be unchanged

Reply:
how can i do this no formatter interference because before committing, the code is the same before but when i committed its the formatter that is doing this 
Can you please suggest me how can i resolve this formatter thing this is what chatgpt suggested 

If you want to prevent your code formatter from automatically formatting your code upon committing, you can take a few steps depending on the tools you're using. Here's how you can do it using Git and popular code formatters like Prettier:

Git Hooks: You can use Git hooks to run scripts before or after certain Git events, such as committing. You can create a pre-commit hook that temporarily disables the formatter before committing. Here's a general idea:

Create a .pre-commit-config.yaml file in your project's root directory (if it doesn't already exist).

Configure the hook to disable the formatter for the staged files using the formatter's command-line options. For example, if you're using Prettier, you can run:

yaml
Copy code
-   repo: https://github.com/pre-commit/mirrors-prettier
    rev: v2.5.1
    hooks:
    - id: prettier
      args: ["--write", "$PRE_COMMIT_HOME/../$PRE_COMMIT_FILES"]
This configuration will use Prettier to format the files, but it will only format the staged files as specified by $PRE_COMMIT_FILES.

Commit the .pre-commit-config.yaml file along with your code.

Configuration Files: Some code formatters, like Prettier, allow you to configure their behavior through configuration files like .prettierrc or prettier.config.js. You can include these configuration files in your project and customize the formatting options to match your preferred style. This way, the formatter will consistently apply the same formatting rules.

Remember, while these methods can help you maintain a consistent code style, it's also a good practice to collaborate with your team and agree on a shared code formatting style that everyone follows. This can help reduce conflicts related to formatting in pull requests.

Reply:
i think you need to disable your formatter in this project from running when you press save. it's a common feature people have when they work on teams and they've agreed on that formatter, but this project is different and your formatter is trying to do the right thing based on it's settings, but it's also making your PR change more than it needs to. i dont know which formatter you have installed on your machine, so i can't give particular instructions. but you could try disabling it entirely while editing this file and then enabling it again once back on your team?

sorry this sounds tedious! but it's also good practice to make the smallest change needed for the feature. changing a whole files single quotes to double quotes, is more than what's needed for the feature. thanks for working some of those formatter changes back, sorry they're in your way.

i'm also on vacation =) will check back here in a couple days